recipe/recipe.yaml

```diff
@@ -0,0 +1,72 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+schema_version: 1
+
+context:
+  version: 0.7.1
+  maturin_version: '>=1.0,<2.0'
+
+package:
+  name: uv-build
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/u/uv-build/uv_build-${{ version }}.tar.gz
+  sha256: c433fc6d92366cc0dc79ed0d003c040d6586e75a5f9630e0588d9869e113720e
+
+build:
+  number: 0
+  # only build for the minimal python version
+  skip: not (match(python, python_min ~ '.*') and is_abi3 == 'true')
+  python:
+    version_independent: true
+  script:
+    file: build-uv-build
+
+requirements:
+  build:
+    - ${{ compiler('c') }}
+    - ${{ compiler('rust') }}
+    - ${{ stdlib('c') }}
+    - cargo-bundle-licenses
+    - if: build_platform != target_platform
+      then:
+        - python
+        - maturin ${{ maturin_version }}
+        - cross-python_${{ target_platform }}
+  host:
+    - maturin ${{ maturin_version }}
+    - pip
+    - python
+    - python-abi3
+  run:
+    - python
+  ignore_run_exports:
+    from_package:
+      - cross-python_${{ target_platform }}
```

Comment:
We should get https://github.com/prefix-dev/rattler-build/issues/1592 fixed ASAP

Reply:
cool: is there anything to be done here?